test/components/gridlinesTests.ts

```diff
@@ -1,63 +1,116 @@
 ///<reference path="../testReference.ts" />
 
 describe("Gridlines", () => {
+  let svg: d3.Selection<void>;
+  let xScale: Plottable.Scales.Linear;
+  let yScale: Plottable.Scales.Linear;
+  let gridlines: Plottable.Components.Gridlines;
 
-  it("Scale ranges are set to the Gridlines dimensions when layout is computed", () => {
-    let svg = TestMethods.generateSVG(640, 480);
-    let xScale = new Plottable.Scales.Linear();
+  beforeEach(() => {
+    svg = TestMethods.generateSVG();
+    xScale = new Plottable.Scales.Linear();
     xScale.domain([0, 10]);
 
-    let yScale = new Plottable.Scales.Linear();
+    yScale = new Plottable.Scales.Linear();
     yScale.domain([0, 10]);
+    gridlines = new Plottable.Components.Gridlines(xScale, yScale);
+  });
 
-    let gridlines = new Plottable.Components.Gridlines(xScale, yScale);
+  it("sets ranges of scales to the Gridlines dimensions when layout is computed", () => {
     gridlines.renderTo(svg);
 
-    assert.deepEqual(xScale.range(), [0, 640], "x scale range extends to the width of the svg");
-    assert.deepEqual(yScale.range(), [480, 0], "y scale range extends to the height of the svg");
+    assert.deepEqual(xScale.range(), [0, TestMethods.numAttr(svg, "width")], "x scale range extends to the width of the svg");
+    assert.deepEqual(yScale.range(), [TestMethods.numAttr(svg, "height"), 0], "y scale range extends to the height of the svg");
 
     svg.remove();
   });
 
-  it("Gridlines and axis tick marks align", () => {
-    let svg = TestMethods.generateSVG(640, 480);
-    let xScale = new Plottable.Scales.Linear();
-    xScale.domain([0, 10]); // manually set domain since we won't have a renderer
+  it("uses the TickGenerators of its Scales", () => {
```

Comment:
Can you change this test to not use `Axes`? Basically, we should be able to render the `Gridlines`, then check that the lines are in the places specified by `xScale.ticks()` and `yScale.ticks()`.